src/app/components/EmailInput/EmailInput.tsx

```diff
@@ -1,11 +1,40 @@
 import { FC } from "react";
 import { EmailInputProps } from "./EmailInput.types";
+import styles from "./EmailInput.module.css";
+
+export const EmailInput: FC<EmailInputProps> = ({ ...props }) => {
+
+  const { testId, required, label, value, error, helperText, placeholder, disabled, onChange } = props;
```

Comment:
Lets destruct these props inside the function args.

Reply:
OK!